package.json

```diff
@@ -35,22 +36,23 @@
   "homepage": "https://github.com/presidenten/wdio-video-reporter#readme",
   "dependencies": {
     "@ffmpeg-installer/ffmpeg": "^1.0.20",
-    "@wdio/allure-reporter": "^7.7.3",
-    "@wdio/reporter": "^7.7.3",
+    "@wdio/allure-reporter": "^8.7.0",
```

Comment:
wdio-video-reporter sends video file to @wdio/allure-reporter
```
import allureReporter from '@wdio/allure-reporter';

allureReporter.addAttachment('Execution video', videoPath, helpers.getVideoFileContentType());
```

Reply:
🤷‍♂️ It was here, we didn't add it :) 
>  or can I just create videos without using the allure reporter?
Makes sense to me. allureReporter is used conditionally in src/index.js if `config.usingAllure` to `addAttachments` and copy generated videos to `config.allureOutputDir`
This can be done, with some ongoing refactoring to remove spagetti around `config.usingAllure`, some side-effects in global config.js and reorganize code in `helpers` - so that potentially allows to do some refactoring and simplify code. 

But that's definitely out of the scope of issue we trying to solve in this PR.

Reply:
I'ld suggest to change that behavior and only run these commands if we can import the allure reporter. People might use it without the Allure reporter and don't want to install this dependency. I am fine to have this as a separate PR, mind creating an issue for it?

Reply:
Ok, I'll try to find time and open a separate PR. It will take some time